### PR TITLE
Test Allow headers for GET and HEAD

### PR DESCRIPTION
**Pull request by kjetilk**

### CODE REVIEW

File: protocol/solid-protocol-test-manifest.ttl [hunk 1]
```diff
@@ -96,3 +96,9 @@ manifest:slug-uri-assignment
   spec:testScript
     <https://github.com/solid/specification-tests/protocol/read-write-resource/post-uri-assignment-slug.feature> .
 
+manifest:read-method-allow
+  a td:TestCase ;
+    spec:requirementReference sopr:server-allow-methods ;
```
Comment (edwardsph) [inline protocol/solid-protocol-test-manifest.ttl:101]:
```suggestion
  spec:requirementReference sopr:server-allow-methods ;
```

File: protocol/read-write-resource/read-method-allow.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Feature: Servers MUST return Allow for GET and HEAD
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()
```
Comment (edwardsph) [inline protocol/read-write-resource/read-method-allow.feature:5]:
```suggestion
```
unnecessary

File: protocol/read-write-resource/read-method-allow.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
+Feature: Servers MUST return Allow for GET and HEAD
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()  
+    * def exampleTurtle = karate.readAsString('../fixtures/example.ttl')
+    * def rdfResource = testContainer.createResource('.ttl', exampleTurtle, 'text/turtle');
+    * def expected = parse(exampleTurtle, 'text/turtle')
```
Comment (edwardsph) [inline protocol/read-write-resource/read-method-allow.feature:8]:
```suggestion
```
unused

File: protocol/read-write-resource/read-method-allow.feature [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    
+  Scenario: GET is supported for containers
```
Comment (edwardsph) [inline protocol/read-write-resource/read-method-allow.feature:10]: The 4 scenario titles appear to have been copied from another test but I think they should refer to the Allow header which is being tested

**Review by edwardsph — APPROVED** [review]
LGTM with suggested changes

File: protocol/read-write-resource/read-method-allow.feature (hunk 4 above)
Comment (kjetilk) [reply protocol/read-write-resource/read-method-allow.feature:8]: Yeah, I can say "allowed" instead, that is more precise.

**Comment by kjetilk** [comment]
argh, forgot to push before I merged.

**Pull request merged**